spec/integration_spec.rb

```diff
@@ -23,6 +23,7 @@ def parse_html_in_email(mail)
   # Rails 5 requires at least ruby version 2.2.2
   if RUBY_VERSION >= "2.2.2"
     rails_apps << RailsApp.new("Rails 5.0.x", 'rails_50', runner: :bin, asset_pipeline: true, digests: true, sprockets3: true)
+    rails_apps << RailsApp.new("Rails 5.0.x (with sprockets 4)", 'rails_50_sprockets_4', runner: :bin, asset_pipeline: true, digests: true, sprockets3: true)
```

Comment:
Could you rename the `sprockets3` options into a `sprockets` config where you set `false`, `2`, `3` or `4`?